medusa/post_processor.py

```diff
@@ -106,10 +106,13 @@ def _get_rel_path(self):
         :rtype: text_type
         """
         if app.TV_DOWNLOAD_DIR:
-            rel_path = os.path.relpath(self.file_path, app.TV_DOWNLOAD_DIR)
-            # check if we really found the relative path
-            if not rel_path.startswith('..'):
-                return rel_path
+            try:
+                rel_path = os.path.relpath(self.file_path, app.TV_DOWNLOAD_DIR)
+                # check if we really found the relative path
+                if not rel_path.startswith('..'):
+                    return rel_path
+            except ValueError:
+                pass
```

Comment:
You mean something like "Couldn't get the relative path, using the parent folder instead" I had in the first commit? We could add it with a bit of refactoring, but don't know how useful that would be - also the error, in this case, is not really something we need to know about in my opinion.